README.md

```diff
@@ -141,7 +141,7 @@ Participation in the Kubernetes community is governed by the [CNCF Code of Condu
 ### SSLError on macOS
 
 If you get an SSLError, you likely need to update your version of python. The
-version that ships with macOS may not be supported.
+the version that ships with macOS may not be supported.
```

Comment:
Repetitive "the" is not needed.

Reply:
revision done. waiting for further feedback.